### PR TITLE
Root style changes

### LINKED ISSUE #73: Redesign background of main Dashboard page
Eliminate the current circuit pattern and replace with stripes and/or floating icons currently in use on main treehacks.com (http://treehacks.com/) site (goal: make this CLASSY!)

- Change default background color of everything, all pages, to light pink #fff7f9 (?) & adjust elements as necessary to ensure nice fit (also adjust navbar)
- add shadow for navbar (copying website of main site)
- make "treehacks" text in logo red instead of black

### LINKED ISSUE #92: Make navbar look like website navbar

### PR DESCRIPTION
**Pull request by lucaspauker**
- Add icons on main page
closes #73

- Fix navbar (make it look like `website`)
closes #92

### CODE REVIEW

**Review by theopolisme — CHANGES_REQUESTED** [review]
This is awesome!!
1. Can we change it so that the navbar is pure white #fff?
2. Also, related, all the text fields/text areas/containers (e.g. the box on the homepage/travel page) should be #fff background as well. I think this way we'll keep all the content much better accented.
3. In navbar, a little more padding between the logo tree and the text (like how it is on https://treehacks.com/) would be great
4. Feels a little weird that drone is going side to side. Maybe should be up/down? Idk though let's discuss in person

Thank you!!!

**Comment by lucaspauker** [comment]
The white-on-pink looks a bit funky imo (maybe look better if colors were more distinct?), but it definitely stands out better.

**Comment by theopolisme** [comment]
Maybe I'm tripping but looks like there's a pink line under the logo - right above the end of navbar/above the shadow?
Screen Shot 2019-10-22 at 5 43 13 PM

File: src/common/Aesthetics.scss [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    min-width: 150px;
+    animation-duration: 2s;
+    animation-iteration-count: infinite;
+    animation-name: updown;
+    animation-timing-function: ease;
+  }
+
+  &.drone {
+    img {
+      animation-name: leftright;
```
Comment (theopolisme) [inline src/common/Aesthetics.scss]:
```suggestion
      animation-name: updown;
```

**Comment by theopolisme** [comment]
@epicfaace looks like snapshots are still failing? would be great to merge asap ❤️

**Pull request merged**